src/sage/rings/padics/padic_template_element.pxi

```diff
@@ -137,7 +147,11 @@ cdef class pAdicTemplateElement(pAdicGenericElement):
                 if x.parent().modulus().change_ring(self.base_ring()) == self.parent().modulus():
                     x = x.polynomial().change_ring(self.base_ring()).list()
                 else:
-                    x = self.base_ring()(x)
+                    if x.polynomial().degree() >= 1:
+                        if self.parent() is self.parent().base_ring():
```

Comment:
Something is wrong here; I think this should be `self.parent() is x.parent().base_ring()`